source/winAPI/constants.py

```diff
@@ -1,11 +1,16 @@
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2022 NV Access Limited
+# Copyright (C) 2024 NV Access Limited
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
 
+import winVersion
 import enum
 
 
+IS_64_BIT: bool = winVersion.getWinVer().processorArchitecture.endswith("64")
+"""True if the system is 64bit"""

```

Comment:
I think we need two flags here. Thoughts @josephsl ?

```suggestion
IS_64_BIT_WINDOWS: bool = winVersion.getWinVer().processorArchitecture.endswith("64")
"""True if the Windows copy is 64bit"""

IS_64_BIT_ARCH: bool = os.environ.get("PROCESSOR_ARCHITEW6432") == "AMD64"
"""True if the system architecture is 64bit"""
```

Reply:
@josephsl perhaps 3?

```suggestion
IS_64_BIT_WINDOWS: bool = winVersion.getWinVer().processorArchitecture.endswith("64")
"""True if the Windows copy is 64bit"""

IS_64_BIT_ARCH: bool = os.environ.get("PROCESSOR_ARCHITEW6432") == "AMD64"
"""True if the system architecture is 64bit"""

IS_64_BIT_NVDA = os.environ.get("PROCESSOR_ARCHITECTURE") == "AMD64"
```

Reply:
```suggestion
IS_64_BIT_WINDOWS: bool = winVersion.getWinVer().processorArchitecture.endswith("64")
"""True if the Windows copy is 64bit"""

IS_64_BIT_NVDA = os.environ.get("PROCESSOR_ARCHITECTURE") == "AMD64"
```